frontend/src/app/AppRouter.tsx

```diff
@@ -90,6 +92,10 @@ export const AppRouter = (): JSX.Element => {
             />
           </Route>
         </Route>
+        <Route
+          path={`/:formId/${PREVIEW_ROUTE}`}
+          element={<PrivateElement isPreview element={<PreviewFormPage />} />}
```

Comment:
feels a little iffy to add such a specific `isPreview` prop to such our route elements haha, should we just add the GovtMasthead directly inside `PreviewFormPage`?